sourcegraph/sourcegraph.proto

```diff
@@ -2169,6 +2169,14 @@ message RepoTreeSearchOp {
 	RepoTreeSearchOptions opt = 2;
 }
 
+message RepoTreeListOp {
+	RepoRevSpec rev = 1 [(gogoproto.nullable) = false];
+}
+
+message RepoTreeListResult {
```

Comment:
Maybe `TreeList` or `RepoTreeList` would be a shorter but still viable name? The current name is fine too.